sky/provision/cudo/cudo_wrapper.py

```diff
@@ -4,29 +4,29 @@
 
 from sky import sky_logging
 from sky.adaptors.cudo import cudo
+import sky.clouds.service_catalog.cudo_catalog_helper as helper
```

Comment:
For the helper, it would be good to be included in `service_catalog.cudo_catalog` directly or placed under `provision.cudo.cudo_utils` to keep the `clouds/service_catalog` folder clean : )